ui/components/Header.js

```diff
@@ -586,6 +586,7 @@ class Header extends React.PureComponent {
       onDrawerCollapse,
       capabilityregistryObj,
     } = this.props;
+    console.log('rendering header');
```

Comment:
```suggestion
```

Reply:
just in case to see if it is still rerendering ..